docs/src/content/docs/commands/ECHO.md

```diff
@@ -0,0 +1,56 @@
+---
+title: ECHO
+description: `ECHO` command in DICEDB is used to print a message
+---
+
+### Syntax
+
+```
+ECHO message
+
+```
+## Parameters
+
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `message` | A string of characters or numbers to print                                | String  | Yes      |
```

Comment:
`message` can be of both type `String/Number` correct?

Reply:
Yes